pillar.example

```diff
@@ -277,7 +283,43 @@ nginx:
   # Passenger configuration
   # Default passenger configuration is provided, and will be deployed in
   # /etc/nginx/conf.d/passenger.conf
+  # Passenger conf can be retrieved by TOFS ( Fallback to nginx.conf )
   passenger:
     passenger_root: /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini
     passenger_ruby: /usr/bin/ruby
     passenger_instance_registry_dir: /var/run/passenger-instreg
+
+  tofs:
+    # The files_switch key serves as a selector for alternative
+    # directories under the formula files directory. See TOFS pattern
+    # doc for more info.
+    # Note: Any value not evaluated by `config.get` will be used literally.
+    # This can be used to set custom paths, as many levels deep as required.
+    files_switch:
+      - any/path/can/be/used/here
+      - id
+      - role
+      - osfinger
+      - os
+      - os_family
```

Comment:
This `files_switch` section is better commented out since we don't really want the extra I/O unless necessary.